_gsocorgs/2019/nikhef.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Nikhef; Dutch national institute for subatomic physics"
+author: "Roel Aaij"
+layout: default
+organization: Nikhef
+logo: logo_Nikhef.jpg
+description: |
+[Nikhef](http://www.nikhef.nl) is the Dutch national institute for
```

Comment:
There are missing spaces in front of the sentence that prevent YAML reading…

Reply:
Fixed to be the same as in #474, I hope this avoids merge conflicts later